synapse/swarm/runtime.py

```diff
@@ -140,8 +140,8 @@ def add(self, tufo):
         return True
 
     def getTufosByPropFrom(self, prop, valu=None, limit=None, mintime=None, maxtime=None, fromtag=deftag):
-
-        dyntask = gentask('getTufosByFrob',prop,valu=valu,limit=limit,mintime=mintime,maxtime=maxtime)
+        timeout = self.maxtime - time.time() if self.maxtime else None
```

Comment:
This could disable the timeout setting if time.time() miraculously equals self.maxtime